wbooks/src/interfaces/book.ts

```diff
@@ -0,0 +1,11 @@
+interface book{
+    id: number;
+    author: String;
+    title: String;
+    genre: String;
+    publisher: String;
+    year: String;
+    imageUrl: String | null;
+}
```

Comment:
Las interfaces van en CamelCase tambien, ojo que String y string no son lo mismo y podes usar el ? para indicar que el valor de imageUrl puede ser udefinded que es lo mas cercano a la realidad cuando manejes la request de back 😄 

```suggestion
interface Book {
    id: number;
    author: string;
    title: string;
    genre: string;
    publisher: string;
    year: string;
    imageUrl?: string;
}
```